ethereum-bytecode-database-proto/ethereum-bytecode-database-proto/proto/ethereum-bytecode-database.proto

```diff
@@ -0,0 +1,163 @@
+syntax = "proto3";
+
+package blockscout.ethereumBytecodeDatabase.v1;
+
+option go_package = "github.com/blockscout/blockscout-rs/ethereum-bytecode-database";
+
+service CreationBytecodeDatabase {
+  rpc VerifySolidityMultiPart(VerifySolidityMultiPartRequest) returns (VerifyResponse) {}
+
+  rpc VerifySolidityStandardJson(VerifySolidityStandardJsonRequest) returns (VerifyResponse) {}
+
+  rpc VerifySolidityMetadata(VerifySolidityMetadataRequest) returns (VerifyResponse) {}
+
+  rpc VerifyVyperMultiPart(VerifyVyperMultiPartRequest) returns (VerifyResponse) {}
+
+  rpc SearchSources(SearchSourcesRequest) returns (SearchSourcesResponse) {}
+}
+
+service DeployedBytecodeDatabase {
+  rpc VerifySolidityMultiPart(VerifySolidityMultiPartRequest) returns (VerifyResponse) {}
+
+  rpc VerifySolidityStandardJson(VerifySolidityStandardJsonRequest) returns (VerifyResponse) {}
+
+  rpc VerifySolidityMetadata(VerifySolidityMetadataRequest) returns (VerifyResponse) {}
+
+  rpc VerifyVyperMultiPart(VerifyVyperMultiPartRequest) returns (VerifyResponse) {}
+
+  rpc SearchSources(SearchSourcesRequest) returns (SearchSourcesResponse) {}
+}
+
+service Sourcify {
+  rpc Verify(VerifySourcifyRequest) returns (VerifyResponse) {}
+}
+
+message Source {
+  // Compiler version used to compile the contract
+  string compiler_version = 1;
+  /// 'settings' key in Standard Input JSON
+  /// (https://docs.soliditylang.org/en/latest/using-the-compiler.html#input-description)
+  string compiler_settings = 2;
+
+  enum Match {
+    MATCH_UNSPECIFIED = 0;
```

Comment:
does sourcify have unspecified match type? what is a situation when we could have this match type?

Reply:
Sourcify does not have unspecified match type, but Grpc should have some default enum variant and partial match does not seem to be the actual default type. Adding an 'unknown' may make it more flexible (https://google.aip.dev/126)